rust/src/sdk/api.rs

```diff
@@ -641,15 +643,13 @@ mod tests {
     fn test_run_client() {
         // check that the client panics when running it without a service
```

Comment:
the comment should be updated to reflect that we now test for a network error instead of a panic.